src/user.ts

```diff
@@ -412,14 +419,15 @@ export class User {
         return await Message.sign(payload, issuer, recipient);
     }
 
-    async getDid() {
+    async createDid() {
```

Comment:
please add return type = string to this function.

also add JSDoc documentation
https://devhints.io/jsdoc